optaplanner-website-root/data/videos.yml

```diff
@@ -12,6 +12,16 @@
 #
 # Order videos by date descending
 
+- youtubeId: A-TyjsHJTNM
+  title: "Introduction to OptaPlanner AI constraint solver"
+  author: dupliaka
+  date: 2022-08-25
+
+- youtubeId: ZEOf_GCdFB8
+  title: "On schedule: Artificial Intelligence plans that meet expectations"
+  author: ge0ffrey
```

Comment:
We should add a comment that Anna worked on this too and support multiple authors over time